pkg/codegen/operations.go

```diff
@@ -621,7 +624,7 @@ func GenerateBodyDefinitions(operationID string, bodyOrRef *openapi3.RequestBody
 		var defaultBody bool
 
 		switch {
-		case contentType == "application/json":
+		case contentType == "application/json" || strings.HasSuffix(contentType, "+json"):
```

Comment:
Do you think you could make a helper function which takes a contentType and determines if it's JSON? I guarantee this will end up getting copy pasted a lot more.